### PR TITLE
Add Xcode themes

### PR DESCRIPTION
**Pull request by anjerodesu**
- AVBushwhacked
- AVTaintedPaint
- AVUnderwater

Remove Multiplex re: https://github.com/kolinkrewinkel/Multiplex/issues/40

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -1231,12 +1231,6 @@
         "screenshot": "https://raw.githubusercontent.com/bugEnding/HHEnumeration-xcode/master/img/after-new.gif"
       },
       {
-        "name": "Multiplex",
-        "url": "https://github.com/kolinkrewinkel/Multiplex",
-        "description": "Simultaneous editing inspired by Sublime Text, for Xcode.",
-        "screenshot": "https://raw.githubusercontent.com/kolinkrewinkel/Multiplex/develop/Meta/Demo.gif"
-      },
-      {
```
Comment (jurre) [inline packages.json]: Why did you remove this?

File: packages.json [hunk 2]
```diff
@@ -1369,6 +1363,24 @@
         "screenshot": "https://raw.githubusercontent.com/paulpilone/xcode-themes/master/AtomLight.png"
       },
       {
+        "name": "AVBushwhacked",
+        "url": "https://raw.githubusercontent.com/anjerodesu/xcode-themes/master/AVBushwhacked.dvtcolortheme",
+        "description": "A dark candy-like theme by Angelo Villegas",
+        "screenshot": "https://raw.githubusercontent.com/anjerodesu/xcode-themes/master/screenshots/AVBushwhacked.png"
+      }
```
Comment (jurre) [inline packages.json]: This needs a `,` here, the JSON is now invalid (and unparsable)

File: packages.json (hunk 1 above)
Comment (anjerodesu) [inline packages.json]: As stated on the comment:
Remove Multiplex re: kolinkrewinkel/Multiplex#40

**Comment by guillaumealgis** [comment]
Thanks @anjerodesu and @jurre ! 👍
